db/migrate/20250217003916_x_summary_large_image_based_on_deprecated_setting.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class XSummaryLargeImageBasedOnDeprecatedSetting < ActiveRecord::Migration[7.2]
+  def up
+    old_setting =
+      DB.query_single(
+        "SELECT value FROM site_settings WHERE name = 'twitter_summary_large_image' LIMIT 1",
```

Comment:
I think we should delete this old row after creating the renamed one.

Reply:
I would like to keep it. Firstly, it can be reverted but also we mark that setting as deprecated and hidden. In a few months we will remove deprecated settings and that will be a good moment to delete it. Does it sound ok to you?

Reply:
Yup, sounds good!